doc_admin/AdminManual.adoc

```diff
@@ -287,7 +285,7 @@ Open Hospital comes with several configuration files called “properties files
 
 We have to specify:
 
-* the *IP address* of the machine where the MySQL Server was installed and containing the Open Hospital database. Replace “localhost” with the IP address (e.g. 192.168.1.100)
+* the *IP address* of the machine where the MySQL Server was installed and containing the Open Hospital database. Replace “localhost” with the IP address (e.g., 192.168.1.100)
```

Comment:
why this comma?

Reply:
In modern American English, a comma should follow both e.g. and i.e. And because they have
both become so commonplace, it is unnecessary to place the abbreviations in italics, even though
they are abbreviated Latin phrases.

See links like: https://jakubmarian.com/comma-after-i-e-and-e-g/ and https://linguaholic.com/linguablog/comma-after-e-g/

Reply:
alright, very interesting debate. Thanks!